### PR TITLE
EHPR-94 | Resolve Database connection

### PR DESCRIPTION
**Pull request by fwpushan**
This PR to fix immediate crash of API lambda.

### CODE REVIEW

**Review by arranfw — APPROVED** [review]

**Comment by arranfw** [comment]
Could you explain what the issue was? Looks like a file was just moved up a level?

**Comment by arranfw** [comment]
Please check https://github.com/bcgov/ehpr2/commit/75afd99d6a3f2f33b30864a5c51993829126bd42 it  may resolve this issue already, we possibly have duplicate DTOs in common

**Comment by fwpushan** [comment]
> Please check [75afd99](https://github.com/bcgov/ehpr2/commit/75afd99d6a3f2f33b30864a5c51993829126bd42) it may resolve this issue already, we possibly have duplicate DTOs in common

In current main branch, building of web app is failing due to this issue. So, my focus is to deploy a crash free app in lambda.

File: apps/api/src/.ormconfig.ts [hunk 1]
```diff
@@ -11,8 +11,8 @@ export const config: ConnectionOptions = {
   username: process.env.POSTGRES_USERNAME,
   password: process.env.POSTGRES_PASSWORD,
   database: process.env.POSTGRES_DATABASE,
-  entities: ['dist/**/*.entity.js'],
-  migrations: ['src/database/migrations/*.ts'],
+  entities: ['./**/*.entity.js'],
```
Comment (hitankar-freshworks) [inline apps/api/src/.ormconfig.ts:14]: are you sure this is working with docker?

**Review by hitankar-freshworks — APPROVED** [review]

**Review by hitankar-freshworks — CHANGES_REQUESTED** [review]
I pretty sure this will break local development

File: apps/api/src/.ormconfig.ts (hunk 1 above)
Comment (fwpushan) [reply apps/api/src/.ormconfig.ts:14]: yes

**Review by hitankar-freshworks — APPROVED** [review]

**Pull request merged**